### DISCUSSION TITLE
How to disable password?

### DISCUSSION THREAD

**Discussion by rotilho** [AdguardTeam/AdGuardHome#5663] [Q&A]
I use SSO in my local server so I don't need to protect AdGuard with a password.

Are there any way to disable AdGuard authentication?

**Comment by YBS-PC** [comment]
maybe it will help, but I'm not sure https://github.com/AdguardTeam/AdGuardHome/issues/1853

**Reply by rotilho** [reply]
Thanks for the reply @YBS-PC . I checked the commit and in my understanding this change enables AdGuard to use glinet as the external authenticator.

In my case the authentication is provided by oauth2-proxy (GCP under the hood) and the validation takes places in the k8s ingress (not in AdGuard).
